tests/dummy/app/templates/components/print-test-attributes.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="attr">{{data-test-attribute}}</div>
```

Comment:
This is only used for testing purposes to see that the `data-test-attribute` is correctly stripped.